requirements.txt

```diff
@@ -0,0 +1,13 @@
+# dev tools
+autopep8
+pylint
+
+# dependencies
```

Comment:
It would be better to add the version of the packages,
you can find the reference in setup.cfg